controllers/work-profiles.js

```diff
@@ -0,0 +1,88 @@
+const WorkProfiles = require('../models/work-profiles.js');
+
+const getAllWorkProfiles = (req, res) => {
+  WorkProfiles.find()
+  .then((WorkProfiles) => res.status(200).json({WorkProfiles}))
+  .catch((error) => res.status(500).json(error));
+};
+
+const getWorkProfileByID = (req, res) => {
+  WorkProfiles.findById(req.params.id)
+  .then((data) =>  res.json({ data }))
+  .catch((error) => res.status(404).json(error))
+};
+
+const getWorkProfilesByName = (req, res) => {
+  WorkProfiles.find({ name: req.params.name })
+  .then((matchedProfiles) => res.json({ matchedProfiles }))
+  .catch((error) => res.status(404).json(error))
+};
+
+const createWorkProfile = (req, res) => {
+  if (!req.body.name || !req.body.description) {    
+    return res.status(400).send('Please include all fields to create a new Work Profile');
+  } else {
+    const workProfile = new WorkProfiles({
+      name: req.body.name,
+      description: req.body.description,
+    });
+    workProfile.save((error, workProfile) => {
+      if (error) {
+        return res.status(500).json({ msg: 'Error while creating a new Work Profile', error });
+      }
+      return res.status(201).json(workProfile);
+    });
+  };
```

Comment:
There's no need to put an `else` in this part if you cover the conditional paths with the first `if` 
```suggestion
  if (!req.body.name || !req.body.description) {
    return res.status(400).send('Please include all fields to create a new Work Profile');
  }
  const workProfile = new WorkProfiles({
    name: req.body.name,
    description: req.body.description,
  });
  workProfile.save((error, workProfile) => {
    if (error) {
      return res.status(500).json({ msg: 'Error while creating a new Work Profile', error });
    }
    return res.status(201).json(workProfile);
  });
```

Reply:
You are right. Done.